CollectionMerge.py

```diff
@@ -0,0 +1,125 @@
+"""
+Merges two collections together by switching references of collection being merged (collection1)
+to the collection being merged into (collection2). If there are catalog number conflicts, a report
+of the catalog numbers is created. If there are no conflicts, then collection merge is preformed.
+
+NOTE: Defaults to saving a report of the catalog numbers that will be merged unless --merge command
+is executed. Deletes the empty collection with the --delete command.
+"""
+import argparse
+import datetime
+import pymysql
+from csvwriter import write_report
+
+def fetch_catalog_numbers(database, collectionid):
+    # Returns list of catalog numbers in collection
+    db_catalog_numbers = database.cursor()
+    db_catalog_numbers.execute("SELECT CatalogNumber FROM collectionobject "
+                               "WHERE CollectionID = '%s'" % collectionid)
+    return [[catalog_number[0]] for catalog_number in db_catalog_numbers.fetchall()]
+
+def check_conflicts(database, collection1, collection2):
+    # Returns list of conflicting catalog numbers and list of catalog numbers from collection1
+    cat_numbers1 = fetch_catalog_numbers(database, collection1)
+    cat_numbers2 = fetch_catalog_numbers(database, collection2)
```

Comment:
also rename `cat_numbers` variables when you rename `collection1` and `collection2`

Reply:
Variables have been renamed for clarity